src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -12,9 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="$(AspNetCoreVersion)" />
```

Comment:
https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel/1.1.1. The idea was that if you include Kestrel, you get hosting for free. We also did this with WebListener. It *might* be ok to drop this now that we have the meta package but we left it in to reduce the chance of a breaking change (since this was intentional).

/cc @Tratcher @DamianEdwards